### ISSUE TITLE
Could not find com.chibde:audiovisualizer:2.2.0.

### ISSUE THREAD

**Issue by Satellite-system** [GautamChibde/android-audio-visualizer#42]
i have used this in my build.gradle(:app) , but getting above error,pls help where i am wrong below is my build.gradle(:app) file

plugins {
    id 'com.android.application'
}

android {
    compileSdk 31

    defaultConfig {
        applicationId "com.assistant.android.apex"
        minSdk 21
        targetSdk 31
        versionCode 1
        versionName "1.0"

        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
    buildFeatures {
        viewBinding true
    }

}

dependencies {

    //TODO 1 : Add Music/Audio Visualizer
    implementation 'com.chibde:audiovisualizer:2.2.0'

    //TODO 2a : Second library added
    debugImplementation 'com.ms-square:debugoverlay:1.1.3'
    releaseImplementation 'com.ms-square:debugoverlay-no-op:1.1.3'
    testImplementation 'com.ms-square:debugoverlay-no-op:1.1.3'

    implementation 'androidx.appcompat:appcompat:1.4.0'
    implementation 'com.google.android.material:material:1.4.0'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.2'
    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.4.0'
    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.4.0'
    testImplementation 'junit:junit:4.13.2'
    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
}

**Comment by JamesLee11** [comment]
I have the same problem. Can you please let us know how to solve this problem?

**Comment by biniya1mmerkin2** [comment]
com.chibde.visualizer.BarVisualizer
i have also the same problem and now i fixed by removing audiovisualizer like above

**Comment by archi2438** [comment]
i have the same problem but the above solution does not work

**Comment by ridayislam402** [comment]
I have also the same problem and now I fixed it. Just add this dependency.

implementation 'io.github.gautamchibde:audiovisualizer:2.2.5'

**Comment by A6uh** [comment]
> com.chibde.visualizer.BarVisualizer i have also the same problem and now i fixed by removing audiovisualizer like above

Yeah it seems like the wiki hasn't been updated for all of the classes besides the SquareBar Visualizer. This solved the issue for me.

**Comment by A6uh** [comment]
> 

If you added io.github.gautamchibde:audiovisualizer:2.2.5 already, try removing com.chibde:audiovisualizer:2.2.0

**Comment by A6uh** [comment]
> > I have also the same problem and now I fixed it. Just add this dependency.
> > implementation 'io.github.gautamchibde:audiovisualizer:2.2.5'
> 
> I added it and it didn't work for me
> 
> when I run or build the app it says : "
> 
> * What went wrong:
>   Execution failed for task ':app:mergeDebugNativeLibs'.
> 
> > Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
> > Could not find com.chibde:audiovisualizer:2.2.0.
> 
> "
> 
> how can I solve it please!.

If you already added
io.github.gautamchibde:audiovisualizer:2.2.5

Then you'll want to remove
com.chibde:audiovisualizer:2.2.0

**Comment by ESO-141** [comment]
> > > I have also the same problem and now I fixed it. Just add this dependency.
> > > implementation 'io.github.gautamchibde:audiovisualizer:2.2.5'
> > 
> > 
> > I added it and it didn't work for me
> > when I run or build the app it says : "
> > 
> > * What went wrong:
> >   Execution failed for task ':app:mergeDebugNativeLibs'.
> > 
> > > Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
> > > Could not find com.chibde:audiovisualizer:2.2.0.
> > 
> > 
> > "
> > how can I solve it please!.
> 
> If you already added io.github.gautamchibde:audiovisualizer:2.2.5
> 
> Then you'll want to remove com.chibde:audiovisualizer:2.2.0
> 
> Keep in mind, a lot of the usage cases in the wiki are also outdated. I opened an issue to correct it here in [issue 52](https://github.com/GautamChibde/android-audio-visualizer/issues/52). You'll probably run into that once you get the dependencies working.



> > > I have also the same problem and now I fixed it. Just add this dependency.
> > > implementation 'io.github.gautamchibde:audiovisualizer:2.2.5'
> > 
> > 
> > I added it and it didn't work for me
> > when I run or build the app it says : "
> > 
> > * What went wrong:
> >   Execution failed for task ':app:mergeDebugNativeLibs'.
> > 
> > > Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
> > > Could not find com.chibde:audiovisualizer:2.2.0.
> > 
> > 
> > "
> > how can I solve it please!.
> 
> If you already added io.github.gautamchibde:audiovisualizer:2.2.5
> 
> Then you'll want to remove com.chibde:audiovisualizer:2.2.0
> 
> Keep in mind, a lot of the usage cases in the wiki are also outdated. I opened an issue to correct it here in [issue 52](https://github.com/GautamChibde/android-audio-visualizer/issues/52). You'll probably run into that once you get the dependencies working.

okay, thanks it worked but and it's fine but when I write it's XML code the XML layout disappears
